src/index.html

```diff
@@ -9,6 +9,73 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+
+            <a href="#" class="nav__link is-active">Apple</a>
+
+          </li>
```

Comment:
Add empty lines only between multiline sibling blocks of HTML. But don't add them between parent and child elements.
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/c43a85c6-a990-4568-b8c2-70623ca92322)